src/Backend/Modules/MediaLibrary/Resources/config/console.yml

```diff
@@ -1,10 +1,9 @@
 services:
-    media_library.console.delete_media_items:
-        class: Backend\Modules\MediaLibrary\Console\MediaItemDeleteAllCommand
+    media_library.console.cache_clear:
```

Comment:
imho this should also be called automatically when the normal cache clear is executed